doc/rocks.txt

```diff
@@ -138,31 +139,41 @@ RocksOpts                                                            *RocksOpts*
+                                                                          (Default: `true` for the best default experience).
         {generate_help_pages?}            (boolean)
-                                                      Whether to re-generate plugins help pages after installation/upgrade. (Default: `true`).
+                                                                          Whether to re-generate plugins help pages after installation/upgrade. (Default: `true`).
         {update_remote_plugins?}          (boolean)
-                                                      Whether to update remote plugins after installation/upgrade. (Default: `true`).
+                                                                          Whether to update remote plugins after installation/upgrade. (Default: `true`).
         {reinstall_dev_rocks_on_update?}  (boolean)
-                                                      Whether to reinstall 'dev' rocks on update
-                                                      (Default: `true`, as rocks.nvim cannot determine if 'dev' rocks are up to date).
+                                                                          Whether to reinstall 'dev' rocks on update
+                                                                          (Default: `true`, as rocks.nvim cannot determine if 'dev' rocks are up to date).
         {enable_luarocks_loader?}         (boolean)
-                                                      Whether to use the luarocks loader to support multiple dependencies (Default: `true`).
+                                                                          Whether to use the luarocks loader to support multiple dependencies (Default: `true`).
         {luarocks_config?}                (table)
-                                                      Extra luarocks config options.
-                                                      rocks.nvim will create a default luarocks config in `rocks_path` and merge it with this table (if set).
+                                                                          Extra luarocks config options.
+                                                                          rocks.nvim will create a default luarocks config in `rocks_path` and merge it with this table (if set).
+        {experimental_features?}          (rocks.ExperimentalFeature[])
+                                                                          List of experimental features to enable.
+                                                                          See |rocks.experimental|.
+
+
+==============================================================================
+experimental features                                       *rocks.experimental*
+
+WARNING: Experimental features are not considered part of the official API.
+They may change or be removed without a major SemVer version bump.
 
```

Comment:
For experimental features to be documented, we need https://github.com/mrcjkb/vimcats/issues/17